test/e2e/specs/importer/importer__import-focused.ts

```diff
@@ -0,0 +1,145 @@
+		it( `Navigate to page setup/import-focused/${ step }?siteSlug=${ siteSlug ?? '' }&from=${
+			from ?? ''
+		}`, async () => {
+			await startImportFlow.startImportFocused( step, siteSlug ?? '', from ?? '' );
+		} );
+	};
+
+	describe( 'Scenario: from `self-hosted` (without JP) to `simple` site', () => {
+		const user = 'simpleSiteFreePlanUser';
+
+		loginAsUser( user );
+		navigateToImportFocusedFlow( 'import', credentials[ user ].testSites?.primary?.url as string );
+
+		it( 'Enter make.wordpress.org', async () => {
+			await startImportFlow.enterURL( 'make.wordpress.org' );
+		} );
+
+		it( 'Should render "Upgrade Plan" screen', async () => {
+			await startImportFlow.validateUpgradePlanPage();
+		} );
+
+		it( 'Should render "Install Jetpack" screen', async () => {
+			await startImportFlow.clickButton( 'Upgrade and migrate' );
+			await startImportFlow.validateInstallJetpackPage();
+		} );
+	} );
+
+	describe( 'Scenario: from `self-hosted` (JP connected) to `simple` site', () => {
+		const user = 'jetpackRemoteSiteUser';
+		const sourceSiteUrl = credentials[ user ]?.testSites?.primary?.url as string;
+		const targetSiteUrl = 'e2eimportflowtesting.wordpress.com';
+
+		loginAsUser( user );
+		navigateToImportFocusedFlow( 'import', targetSiteUrl );
+
+		it( `Enter jetpack connected source site: ${ sourceSiteUrl }`, async () => {
+			await startImportFlow.enterURL( sourceSiteUrl );
+		} );
+
+		it( 'Should render "Upgrade Plan" screen', async () => {
+			await startImportFlow.validateUpgradePlanPage();
+		} );
+
+		it( 'Should redirect to "Checkout" page', async () => {
+			await startImportFlow.clickButton( 'Upgrade and migrate' );
+			await startImportFlow.validateCheckoutPage();
+		} );
+	} );
+
+	describe( 'Scenario: from `self-hosted` (JP connected) to `atomic` site', () => {
+		const user = 'jetpackRemoteSiteUser';
+		const sourceSiteUrl = credentials[ user ]?.testSites?.primary?.url as string;
+		const targetSiteUrl = 'e2eimportflowatomictesting.wordpress.com';
+
+		loginAsUser( user );
+		navigateToImportFocusedFlow( 'import', targetSiteUrl );
+
+		it( `Enter jetpack connected source site: ${ sourceSiteUrl }`, async () => {
+			await startImportFlow.enterURL( sourceSiteUrl );
+		} );
+
+		it( 'Should render "Migration Ready" screen', async () => {
+			await startImportFlow.validateMigrationReadyPage();
+		} );
+	} );
+
+	describe( 'Scenario: Move to WPCOM plugin: from `self-hosted` to `simple` site', () => {
+		const user = 'jetpackRemoteSiteUser';
+		const sourceSiteUrl = credentials[ user ]?.testSites?.primary?.url as string;
+
+		loginAsUser( user );
+		navigateToImportFocusedFlow( 'migrationHandler', '', sourceSiteUrl );
+
+		it( 'Should render "Site Picker" screen', async () => {
+			await startImportFlow.validateSitePickerPage();
+		} );
+
+		it( 'Should select the second site and press `Continue` on modal prompt', async () => {
+			await page.locator( 'button:text("Select this site")' ).nth( 1 ).click();
```

Comment:
Let's use user-facing attributes for locating when possible:

```suggestion
			await page.getByRole( 'button', { name: "Select this site" } ).nth( 1 ).click();
```

See: https://playwright.dev/docs/best-practices#prefer-user-facing-attributes-to-xpath-or-css-selectors